k8s/agent-operator/pom.xml

```diff
@@ -81,12 +81,6 @@
             <artifactId>jaxb-impl</artifactId>
         </dependency>
 
-        <!-- required by the fabric8 client, but not included automatically for some reason -->
```

Comment:
I think the dependency is still needed, without it fabric8 fails during the init.

Reply:
oops

Reply:
dixed